SAPHana/ra/SAPHanaController

```diff
@@ -349,6 +350,28 @@ function version() {
     fi
 }
 
+# function: get_action_timeout
+# output:  action timeout
+#
+# timeOut = max(3600, 95%(actionTimeOut))
+#
+function get_action_timeout() {
+    super_ocf_log info "FLOW $FUNCNAME ($*)"
+    # $OCF_RESKEY_CRM_meta_timeout is the timeout of the current running action in ms
+    local actionTimeOut="${OCF_RESKEY_CRM_meta_timeout:-$HANA_STD_ACTION_TIMEOUT}"
```

Comment:
OCF_RESKEY_CRM_meta_timeout is in milliSeconds (3.600.000) while HANA_STD_ACTION_TIMEOUT is in seconds (3600) So we could not use a value in [s] for a variable which counts in [ms]. Or we first need to divide OCF_RESKEY_CRM_meta_timeout by 1000 to translate from [ms] to [s].

Reply:
right, changed after tests, but forgot to commit. Sorry and thanks for pointing that out.